content/pt-br/docs/tasks/access-application-cluster/access-cluster.md

```diff
@@ -0,0 +1,269 @@
+
+Se o aplicativo for disponibilizado como um pod no cluster, consulte a [próxima seção] (#acessar-o-api-deum-pod).
+
+### Biblioteca Python client
+
+Para usar o [cliente Python](https://github.com/kubernetes-client/python), execute o seguinte comando:
+`pip install kubernetes`. Consulte [a página Python Client Library](https://github.com/kubernetes-client/python)
+para obter mais opções de instalação.
+
+O cliente Python pode usar o mesmo arquivo [kubeconfig](/docs/concepts/configuration/organize-cluster-access-kubeconfig/)
+como a CLI do kubectl faz, para localizar e autenticar ao apiserver. Veja esse
+[exemplo](https://github.com/kubernetes-client/python/tree/master/examples).
+
+### Outras bibliotecas
+
+Existem [bibliotecas de clientes](/docs/reference/using-api/client-libraries/) para acessar a API de outras linguagens.
+Consulte a documentação de outras bibliotecas para saber como elas se autenticam.
+
+## Acessando a API a partir de um pod
+
+Ao acessar a API de um pod, a localização e a autenticação
+para o servidor de API são um pouco diferentes.
+
+Consulte [Acessando a API a partir de um pod](/docs/tasks/run-application/access-api-from-pod/)
+para obter mais detalhes.
+
+## Acessando serviços em execução no cluster
+
+A seção anterior descreve como se conectar ao servidor da API do Kubernetes.
+Para obter informações sobre como se conectar a outros serviços em execução em um cluster do Kubernetes, consulte
+[Acessando cluster serviços](/docs/tasks/access-application-cluster/access-cluster-services/).
+
+## Solicitação de redirecionamentos
+
+Os recursos de redirecionamento foram descontinuados e removidos. Em vez disso, use um proxy (veja abaixo).
+
+## Tantos proxies
+
+Há vários proxies diferentes que você pode encontrar ao usar o Kubernetes:
+
+1. O [kubectl proxy](#directly-accessing-the-rest-api):
+
+   - é executado no computador de um usuário ou em um pod
+   - configure proxy de um endereço localhost para o apiserver do Kubernetes
+   - cliente para proxy usa HTTP
+   - proxy para o apiserver usa HTTPS
+   - localiza o apiserver
+   - adiciona headers de autenticação
+
+2. O [apiserver proxy](/docs/tasks/access-application-cluster/access-cluster-services/#discovering-builtin-services):
+
+   - é um bastião incorporado ao apiserver
+   - conecta um usuário fora do cluster aos IPs do cluster que, de outra forma, poderiam não ser acessíveis
+   - é executado no processo do apiserver
+   - cliente para proxy usa HTTPS (ou http se o apiserver estiver configurado dessa forma)
+   - O proxy para o alvo pode usar HTTP ou HTTPS, conforme escolhido pelo proxy usando as informações disponíveis
+   - pode ser usado para acessar um Nó, Pod ou Serviço
+   - faz o load balancing quando usado para acessar um serviço
+
+3. O [kube proxy](/docs/concepts/services-networking/service/#ips-and-vips):
+
+   - é executado em cada nó
+   - proxy de UDP e TCP
+   - não entende HTTP
+   - fornece load balancing
```

Comment:
```suggestion
   - fornece balanceamento de carga
```

Reply:
Fixed and noted for future localization work